dagshub/streaming/filesystem.py

```diff
@@ -362,6 +362,15 @@ def install_hooks(self):
         os.chdir = self.chdir
         self.__class__.hooked_instance = self
 
+    @classmethod
+    def uninstall_hooks(cls):
+        if hasattr(cls, f'_{cls.__name__}__unpatched'):
+            io.open = cls.__unpatched['open']
+            os.stat = cls.__unpatched['stat']
+            os.listdir = cls.__unpatched['listdir']
+            os.scandir = cls.__unpatched['scandir']
+            os.chdir = cls.__unpatched['chdir']
```

Comment:
`install_hooks()` also reassigns other `open`s and `stat`s etc (see line 358), we should probably clean up all of them. 


```suggestion
            io.open = builtins.open = _pathlib.open = cls.__unpatched['open']
            os.stat = _pathlib.stat = cls.__unpatched['stat']
            os.listdir = _pathlib.listdir = cls.__unpatched['listdir']
            os.scandir = _pathlib.scandir = cls.__unpatched['scandir']
            os.chdir = cls.__unpatched['chdir']
```